src/effects/VST3/VST3Effect.cpp

```diff
@@ -942,11 +942,7 @@ void VST3Effect::ExportPresets(const EffectSettings &) const
       return;
    }
 
-   if(!Vst::PresetFile::savePreset(
-      fileStream,
-      FUID::fromTUID (mEffectClassInfo.ID().data()),
-      mEffectComponent.get(),
-      mEditController.get()))
+   if (!VST3Wrapper::SavePreset(fileStream, FUID::fromTUID(mEffectClassInfo.ID().data())))
```

Comment:
The wrapper base class already hass `mEffectClassInfo` so this call and the call to `LoadPreset` can pass just one argument.